From c21509585749889f357d56326c5b82bb919292c2 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <jkysela@redhat.com>
Date: Wed, 3 Sep 2014 07:17:09 -0400
Subject: [alsa] hda: Add missing mixer widget for AD1983

Message-id: <140972823573-83-patchset-jkysela@redhat.com>
Patchwork-id: 90783
O-Subject: [RHEL 7.1 PATCH 082/442] ALSA: hda - Add missing mixer widget for AD1983
Bugzilla: 1112200
RH-Acked-by: John Linville <linville@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Ivan Vecera <ivecera@redhat.com>

The mixer widget on AD1983 at NID 0x0e was missing in the commit
[f2f8be43c5c9: ALSA: hda - Add aamix NID to AD codecs].

Bugzilla: https://bugzilla.kernel.org/show_bug.cgi?id=70011
Cc: <stable@vger.kernel.org>
Signed-off-by: Takashi Iwai <tiwai@suse.de>

(picked from kernel: c7579fed1f1b2567529aea64ef19871337403ab3)
rhbz#1112200
Signed-off-by: Jaroslav Kysela <jkysela@redhat.com>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/sound/pci/hda/patch_analog.c b/sound/pci/hda/patch_analog.c
index 50b2427..195cd62 100644
--- a/sound/pci/hda/patch_analog.c
+++ b/sound/pci/hda/patch_analog.c
@@ -498,6 +498,7 @@ static int patch_ad1983(struct hda_codec *codec)
   return err;
  spec = codec->spec;
 
+ spec->gen.mixer_nid = 0x0e;
  spec->gen.beep_nid = 0x10;
  set_beep_amp(spec, 0x10, 0, HDA_OUTPUT);
  err = ad198x_parse_auto_config(codec, false);
-- 
1.7.1